metadata.json

```diff
@@ -72,7 +72,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.0.0 < 7.0.0"
```

Comment:
Think you need to bump the minimum version of `concat` too?  (To the version you added support for Deferred to) :)

Reply:
Right :D /me fixing.